HappyApp.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+from time import sleep 
+from config import MessageType, status,_Hour, day, _1PM, _8AM, _8PM
+from functions import isToday, scramble, alert
+from queue import Queue
+import readJSON
+from mail import Mail
+
+class HappyApp:
+    def __init__(self,):
+        self.__roll()
+        port, sender_email, sender_pw = readJSON().credentials()
+        self.mail = Mail(port, sender_email, sender_pw)
+
+    def __roll(self):
+
+        # Read names
+        keva_names  = list(readJSON().keva().keys())
+        sadir_names = list(readJSON().sadir().keys())
+
+        sadir_names_rnd = scramble(sadir_names)
+
+        sadir_pairs = [(sadir_names_rnd[i],
+                       sadir_names_rnd[i+1])
+                       for i in range(0, len(sadir_names_rnd) - 1, 2)]
+
+        if len(sadir_names_rnd) % 2 != 0:
```

Comment:
this can result in unfair rounds since there is no consideration for past round credits, perhaps we should add a credit class that assigns a value to each member of section 14, and then assign the person with the least credit to enter the round for the second time, in case of an uneven number of soldiers.

Reply:
You are right, i will need to make changes.